providers/src/airflow/providers/pagerduty/CHANGELOG.rst

```diff
@@ -27,6 +27,20 @@
 Changelog
 ---------
 
+3.8.1
+.....
+
+Misc
+~~~~
+
+* ``Move bash operator to Standard provider (#42252)``
+* ``Purge existing SLA implementation (#42285)``
```

Comment:
Hmm this one is because https://github.com/apache/airflow/pull/42285/files#diff-8fa4fd0e9c359df89fde6c3043b13aa7d36d9f63aebc1744bffc43e00b158886L26 - i.e. removal of `sla_miss` documentation. This is a small thing and probably not worth reverting, but it looks like the providers affected (pagerduty, slack, smtp) will miss the `sla_miss` documetnation. Not a big deal, would be better to have a note instead that it wil only work in 2.0 - @ferruzzi  -> not sure if this is worth the effort.